reconcile/quay_membership.py

```diff
@@ -67,12 +67,19 @@ def fetch_current_state(quay_api_store):
         if not teams:
             continue
         for team in teams:
-            members = quay_api.list_team_members(team)
-            state.add({
-                'service': 'quay-membership',
-                'org': org_key,
-                'team': team
-            }, members)
+            try:
+                members = quay_api.list_team_members(team)
+            except QuayTeamNotFoundException:
+                logging.warning("Attempted to list members for team %s/%s, "
+                                "but it doesn't exist", org_key.org_name, team)
```

Comment:
Could you include org_key.instance in the log?

Reply:
Good call, I'll fix that!